CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.3.6] - 2019-01-24
+### Changed
+- Modified `cobertura-maven-plugin` configuration in order to be not inherited by submodules.
+- Updated `jacoco-maven-plugin` version to 0.8.2 (was 0.8.1)..
```

Comment:
Remove the extra period at the end, same for next line and line 13.

Reply:
done